### PR TITLE
Php-cs-fixer integration

### PR DESCRIPTION
**Pull request by Billz95**
See https://github.com/prettier/plugin-php/issues/939#issuecomment-459549609 for the motivation.
This PR gives a recipe for customised Fixer that uses `prettier/plugin-php` to work in `php-cs-fixer`
Thanks @aboyton for the initial idea for developing this Fixer.

### CODE REVIEW

File: docs/recipes/php-cs-fixer/README.org [hunk 1]
```diff
@@ -0,0 +1,31 @@
+* Prettier integration to php-cs-fixer
```
[the remark this answers is not in the document]
Comment (Billz95) [reply docs/recipes/php-cs-fixer/README.org]: thx for suggestion, fixed.

File: docs/recipes/php-cs-fixer/PrettierPHPFixer.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    public function supports(SplFileInfo $file) {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    private function applyFix(SplFileInfo $file, Tokens $tokens) {
+        $tmpFile = $this->getTmpFile($file);
+        exec("yarn exec -- prettier --write --brace-style=1tbs $tmpFile");
```
Comment (czosel) [inline docs/recipes/php-cs-fixer/PrettierPHPFixer.php]: I'd stick with the default configuration (no options) in this example.

**Comment by czosel** [comment]
Apart from the nitpick, this looks great! Thanks for contributing :+1:

File: docs/recipes/php-cs-fixer/PrettierPHPFixer.php (hunk 2 above)
Comment (Billz95) [reply docs/recipes/php-cs-fixer/PrettierPHPFixer.php]: thx for suggestion, yes this makes much more sense when this is going public 😄

**Comment by czosel** [comment]
@evilebottnawi ready for final review.

**Pull request merged**